libs/form-builder/src/components/SplitRender.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { SplitColumn } from '@daohaus/ui';
+
+import { FieldLego } from '../types';
+import { FormBuilderFactory } from './FormBuilderFactory';
+
+type SplitColumnProps = {
+  id: string;
+  disabled?: boolean;
+  rows: { rowId: string; left: FieldLego; right: FieldLego }[];
+};
+
+export const SplitColumnLayout = ({ rows, ...props }: SplitColumnProps) => {
+  return (
+    <SplitColumn
+      rows={rows.map(({ left, right, rowId }) => {
+        return {
+          rowId,
+          left: <FormBuilderFactory {...left} {...props} spacing={false} />,
```

Comment:
Are there potential concerns around implementing the factory in two places? I don't have knowledge of state management.

Reply:
Let's talk through the this. From my understanding, since the factory is stateless it should be fine. Recursive rendering is a pattern is a pattern I'm hoping to user throughout the builders. So it would be good to highlight any challenges we foresee,